readthedocs/audit/models.py

```diff
@@ -157,6 +178,11 @@ def save(self, **kwargs):
         if self.project:
             self.log_project_id = self.project.id
             self.log_project_slug = self.project.slug
+            organization = self.project.organizations.first()
+            if organization:
```

Comment:
Un-related to this PR: now that we have Organizations in .org, we can migrate the `ManyToMany` from Organization model into a ForeignKey in the Project model and avoid all these `.first()` checks we have all around the code.